templates/item-info.widget

```diff
@@ -152,64 +152,84 @@ function editItemInfo(itemUid) {
   switchSection("#item-" + itemUid + " .item-info", "editing");
 }
 
+CSS
+------------------------------------------------------------
+.formLabel {
+  display: block;
+  margin-bottom: 5px;
+  margin-top: 15px;
+}
+
+.form-group {
+  margin-top: 15px;
+  margin-bottom: 5px;
+}
+
+.form-btn-group {
+  margin-top: 20px;
+  margin-bottom: 5px;
+}
+
+.save {
+  margin-right: 20px;
+}
 
 HTML: item-info-edit-form
 ------------------------------------------------------------
 {{! "autocomplete=off" everywhere: http://stackoverflow.com/q/8311455 }}
-<form onsubmit="submitItemInfo('{{item.uid}}', this); return false;">
-  <label>
-    Name<br>
-    <input name="name" value="{{item.name}}"
-           type="text" autocomplete="off">
+<form class="form-group" onsubmit="submitItemInfo('{{item.uid}}', this); return false;">
+  <label class="formLabel" for="name">
```

Comment:
If all labels have class `formLabel`, isn't it easier to just add a style for e.g. `.item-info-edit-form > label`?